mitodl/micromasters
# Add courseware_backend to CourseRun

#### Pre-Flight checklist

#### What are the relevant tickets?
Fix #4991 

#### What's this PR do?
Add courseware_backend to CourseRun

#### How should this be manually tested?
Run migrations, Check that all existing course runs have a new field with the default value, nothing should break.
